### ISSUE TITLE
Invitation for Task Force for New Catalog Record Search UI

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#5324]
Hi all, we are in the process of forming a task force to develop recommendations for the new Catalog Record Search interface. If you are interested in participating, please fill out this When2Meet poll so we can set up a time: https://www.when2meet.com/?17892746-dRDRH. I know everyone has a lot going on, but these are major changes to an essential tool, so ideally we should meet up at least once before the holidays so that we can get @dustymc the feedback he needs to develop this page for all our user communities.

**Comment by Jegelewicz** [comment]
From today's meeting

- [ ] Team to update [category sheet](https://docs.google.com/spreadsheets/d/1XOUYr7dqKx_koobGg1tQu98hi_S_SHJMqJsjcuQf8YA/edit#gid=1478594408)
- [ ] Dusty to add the categories to the interface
- [ ] Let the critique begin (Dusty don’t DO anything until the team has worked through conflicting requests)
- [ ] Meet again next week and with address comments or whatever we are able to do given progress to that date

**Comment by dustymc** [comment]
I think the 'basic' category needs to go from the spreadsheet - those will just be integrated in with the rest of the identifiers-or-whatever.

I think 'default' (basic, whatever) makes a good subcategory for the initial/public view, and serves the purpose in a categorized form.

One (simple, I hope) item that hasn't yet been discussed is the name/url of the page. That will need solidified before this page becomes the default, and I assume we don't want to reuse the current``Specimen...`` terms. I suppose I'll stick with ``/cat_records.cfm`` or let my password manager choose (``&fcevZmA#63C&0j11t5D@TkGz9ZA1m9Hw1oTG5TI5gCkA$3Ifl.cfm`` is nice) or something of the sort if nobody has clever ideas before whenever I need them....

**Comment by Jegelewicz** [comment]
> One (simple, I hope) item that hasn't yet been discussed is the name/url of the page. 

search

**Comment by dustymc** [comment]
I just rebuilt the page with basically no regard for the spreadsheet....

https://arctos-test.tacc.utexas.edu/cat_records.cfm

categories are

Identifiers
Identifications
Catalog Record
Parts
Usage
Event
Locality
Spatial

which maybe seems about right?? Easy enough to shuffle things, but I would like to get this somewhat solidified ASAP, and hopefully whatever I did helps the conversation move.

If that seems somewhat reasonable and the form is closer to what's desired, I'll work on making the layout a little less East Berlin (so many gray squares...).

This does two things, which are really different sides of the same coin:

* The form isn't as user-customizable as it was a few minutes ago. A user who doesn't give a crap about (whatever) still has to scroll by the giant empty 
 # WHATEVER
section, and
* The form is more predictable/controllable - we could do more than variable-width grid and devolving tables with this, although that might be in 'hire a CSS person' territory.

**Comment by dustymc** [comment]
I've sort of spiraled in on what I think is a nice balance of ease of use (turn on all category or category-->subcategory items with a click or two) and fully customizable (turn one of the bundled items off). 

The search form now has sections, and each of them has a customize button.

<img width="219" alt="Screenshot 2022-12-07 at 10 12 20 AM" src="https://user-images.githubusercontent.com/5720791/206262627-715cd158-22f5-4c6b-b5f8-ab1004954648.png">

Clicking it will take you to that category, where you can 

* turn everything on or off
* turn subcategories on or off
* turn individual items on or off

Please let me know as soon as possible if that approach isn't workable for some reason.

Items attributed to a category should be in that pane, but that's hard-coded so please let me know (or update the category) if you find a misplaced item.

The form will initialize (for public users, users who haven't yet customized, or users who've just turned everything off) with all subcategory='basic' items on. A category can have any number (including zero) of 'basic' items.

The order of the categories in the spreadsheet should match the form, and the default_order column has been updated to match the form (and that loaded to test).

Category order can be changed, but it'll need coordinated with code.

Major TODOs:

* Subcategory classification and order can be cleaned up.
* Display can be cleaned up.

**Comment by dustymc** [comment]
Unless there's some immediate objection, I think I'll merge 'usage' into 'catalog_record' - I can't quite see where the border is, which suggests there is not one.

**Comment by ebraker** [comment]
Is this test link for the new basic search or the new customize search form? If it is for the new customize form, then great.

If it is for the new basic search, I think we need to focus on expanding and collapsing the fields vs. adding and removing fields.  Adding and Removing takes work to scroll through the giant lists of options and it is a major step back from the functionality we currently have (scroll through a giant list, select what you want, then fill in values, and maybe do that a few more times over if you want to add fields from different categories). I think it is critical that we be able to view each category with a few basic fields visible, and then have an "expand"/"see more options" action to view ALL of the existing fields that are currently present on the classic search page. Adding fields is completely overwhelming, especially to someone new to Arctos who doesn't even know what the options are.

(Please ignore this comment if https://arctos-test.tacc.utexas.edu/cat_records.cfm is the new customize form - if it is, I think it is much more legible than the previous customize version) 

> Unless there's some immediate objection, I think I'll merge 'usage' into 'catalog_record' - I can't quite see where the border is, which suggests there is not one.

I agree that it makes sense to combine if this is for the customize form. If this is the basic form, i would err on breaking up in categories into more palatable chunks (combining makes 94 fields to select from under the catalog record category).

**Comment by dustymc** [comment]
Yes that's the correct link.

>work to scroll through the giant lists 

Screenshot? There should be no/minimal scrolling necessary: add/remove, all/none, save, done.

> able to view each category with a few basic fields visible,

Set subcategory to 'basic' and this happens. 

>view ALL of the existing fields that are currently present on the classic search page

I still think this is not palatable. Very few users could possibly have any interest in accession agency, 'genus' is not what most users will think, etc.

**Comment by dustymc** [comment]
>94 fields to select from under the catalog record category

Maybe we need to (somehow) do more with subcategories, somehow? Eg a couple of those fields are broken-out attributes for https://github.com/ArctosDB/arctos/issues/5341, I think they need to land with the 'normal' attribute stuff (currently catalog_record) but they're also specialist terms which most users probably don't care about/want to see. That sort of thing might happen with a couple hundred more attributes. I suppose I could add some sort of subcategory shortcut but that could get out of control too.

**Comment by dustymc** [comment]
Here, I hope, is what I failed to say in the two above comments.

1. Let's add enough to 'basic' that most users won't need to customize (but not so much that everyone has to scroll, either)
2. Can we do more with profiles? We've got to create them before we can go very far with this, but I have the recipe for two so there's a start. Maybe some sort of profile selector somewhere on the page. I have active requests for uam_eh and uam_art, but rather than specific collections those could include 'tissue-centric mammal search' or WHATEVER.

**Comment by campmlc** [comment]
Would profiles be found with the "filter for tissues" or "filter for art collections" or "filter for earth sciences"? table we discussed?

**Comment by dustymc** [comment]
>have to scroll through

But you don't: click "All" and "Save." Can we somehow address that in the UI, while letting users know that there are finer-scaled options (by showing them the subcategories)?

**Comment by dustymc** [comment]
>intimidated

I'm thinking most users should not need to customize (and maybe something clever with profiles ups "most"), but EVERYONE will be intimidated by seeing everything. IDK, playing with more buttons now....

> "culture of origin" and "culture of use" may be better placed under Identification(?)

They're record attributes, I can't imagine how scattering one thing across multiple categories isn't confusing??

**Comment by dustymc** [comment]
>equivalent of clicking 

AHA, thanks! Ghost-clicking implemented.

Turning everything on might still be intimidating, but the not-quite-button that giveth can also taketh so ....

**Comment by ewommack** [comment]
For the result table, is there a way to add columns there? Thinking back to when we add fields to search by, it would be great to be able to sort by some of those columns when we get the results. Or like I'd like to add sex to as a column so I can sort by and see grouped together all of the males, and then all of the females. I can search on attribute -> sex -> but then I have to enter a value to search that attribute.

I like the colored boxes. Should we think about branding, and use different shades of Arctos blue?

When I click the customize button, all of the boxes are checked, instead of just starting with the ones I had previously. Now I have to go back and uncheck a bunch of things. I can hit uncheck all, but then I have to remember what I need to have checked for a basic search, or what I had checked before.

**Comment by dustymc** [comment]
>result table, is there a way to add columns there?


<img width="166" alt="Screenshot 2022-12-07 at 1 54 08 PM" src="https://user-images.githubusercontent.com/5720791/206306054-08b19dcd-9157-409b-82e4-948dcb0bcf58.png">

> sort by 

click headers

<img width="193" alt="Screenshot 2022-12-07 at 1 54 54 PM" src="https://user-images.githubusercontent.com/5720791/206304907-edf21f6c-b280-42b2-be01-76d8c2bf326b.png">

>search on attribute -> sex -> but then I have to enter a value to search that attribute.

Different Issue, this is just reusing all back-end stuff.  See also https://github.com/ArctosDB/arctos/issues/5341, that's probably a good candidate for a dedicated search as well.

>use different shades of Arctos blue?

I've literally been using random color pickers, except then they keep spitting out purple and I have @lkvoong pick a random color (which is inevitably a shade of green...). Yes some sort of color palette would be fabulous, and deserving of its own issue. See also https://github.com/ArctosDB/arctos/issues/4874

>When I click the customize button, all of the boxes are checked

I need screenshots or something. You should be seeing...


<img width="165" alt="Screenshot 2022-12-07 at 1 59 37 PM" src="https://user-images.githubusercontent.com/5720791/206305757-2b951621-279d-4082-96f6-eab2af32d92f.png">

... which lets you check and uncheck whatever you want, starting with whereever you left off, and for each section....


<img width="342" alt="Screenshot 2022-12-07 at 2 01 05 PM" src="https://user-images.githubusercontent.com/5720791/206305948-e330bf3e-8fe9-4cf1-a29a-3cccdea4dbb1.png">

... which does what it says it'll do and is hopefully intuitive??

**Comment by ewommack** [comment]
It reset itself after I backed out of everything and restarted the page, but for a while any time I clicked any of the customize buttons this is what I saw right when the window opened. I can try and recreate the error if you want?
![Untitled_arctos customization craziness](https://user-images.githubusercontent.com/18562884/206307461-11acc851-b5f1-4fe1-b955-a5dc5ad00092.jpg)


I'm not logged in and I can't find the Customize Results tab:
![results can't customize](https://user-images.githubusercontent.com/18562884/206307590-1f92e343-f13a-46d8-8bfa-8397b4e29376.jpg)

**Comment by ewommack** [comment]
@mkoo -> where are we storing those Arctos branded colors? From that nifty letterhead you created...

**Comment by dustymc** [comment]
>t reset itself 

I'm changing things, you probably got caught up in that.

>not logged in and I can't find the Customize Results 

It's not available without login, maybe-yet.

**Comment by ewommack** [comment]
> It's not available without login, maybe-yet.

Sadness.

**Comment by dustymc** [comment]
>search

Can't say I'm terribly thrilled, lots of other things can be searched and maybe someday we'll have some sort of elastic "global search" and etc., but I'm not hearing any better idea so here we go...

**Comment by ewommack** [comment]
I think the grays could work, and then all of your default and visited link colors can be seen well. I do like the use of the blues though. It does make it more consistent with the Arctos colors

**Comment by dustymc** [comment]
>couple of other options
>the grays could work

Not sure if this is the right grays but I'm not sure about the blues either, and it's (obviously, I hope) easy enough to swap things around in this architecture, so...

<img width="1709" alt="Screenshot 2022-12-12 at 4 38 59 PM" src="https://user-images.githubusercontent.com/5720791/207198000-93a64c79-effe-4f30-b3a4-fd804840afd4.png">

**Comment by ewommack** [comment]
Oooh, I like how that looks. It is easy to see the different rows and groups.

**Comment by ewommack** [comment]
Ok basic question -> what is the difference between the two search customization save columns? Even when not logged in I can click save in either column, but I'm not sure what difference it does?
![Search field question](https://user-images.githubusercontent.com/18562884/207404360-03425250-87b5-4fc5-ab8f-6d000bdd0093.png)

**Comment by dustymc** [comment]
first columns is "category" so you can turn all part-stuff on with a click.

second is 'subcategory' so you can turn on all "part-part" and no "part-container" on with a click.

So basically just shortcuts.

**Comment by dustymc** [comment]
meeting todo

* add section at top, repeat buttons from bottom plus number of records

**Comment by ewommack** [comment]
> first columns is "category" so you can turn all part-stuff on with a click.

> second is 'subcategory' so you can turn on all "part-part" and no "part-container" on with a click.

Ok, that definitely needs a help button or something. I'm not sure I'd ever figure that, so maybe add it to the new text to the Customize?

**Comment by ebraker** [comment]
Here's some padding- white border around query block with an outline. I think it brings some of the old feel back!

![image](https://user-images.githubusercontent.com/11336485/207444289-05acba60-f788-4713-bbab-c7b34ba9fbb7.png)

**Comment by mkoo** [comment]
Emily and I worked up some mockups and palette hex values based on our meeting yesterday but I am unsure the best way to share the pptx and other info. @ebraker I think I will download the office365 file and post as a gslideshow (it will count against my allocation then, and I will only share the first half of the pres) -- ok?

@dustymc we have also ahve another doc with the suggested colors for specific CSS classes.... so more soon

**Comment by ebraker** [comment]
@mkoo that works for me!

**Comment by mkoo** [comment]
ok [here's the whole slide deck](https://docs.google.com/presentation/d/1CKE2lwPzCllZLgcjTUMGurQHdPPsvNb5/edit#slide=id.p1)-- feel free to add notes @ebraker. @dusty I will still add a set of class and proposed hexcodes soon, but you can see some of our thinking here

all, nothing is set here and we can explain more of our alternate options -- I also have some new thoughts on the main menu bar: I do think we have to return to grays/ neutral palette now that I've checked out a few more guid pages.

**Comment by dustymc** [comment]
> main menu bar... have to return to grays/ neutral palette now that I've checked out a few more guid pages.

Not sure I'm fully understanding and what I think it going to happen is subject to change, but see https://github.com/ArctosDB/arctos/issues/5333 and https://github.com/ArctosDB/internal/issues/211: my intention is that the header/navbar/menu-thingee will be relatively small (eg small logo and one row of links), at the very top of the page, and WILL NOT be changed on GUID pages (or anywhere else). The current practice of cramming data in the header, and carrying around a huge header so that there's space to jam things in (except when there's not, which is increasingly common), is - uhh, let's call it "suboptimal." The header/navbar should be something that can be safely ignored unless the user is wanting to go somewhere else - not something that contains the most critical data! ANYWAY - some neutral color which isn't going to be too clashy with whatever gaudy color the next collection might bring to Arctos seems like an outstanding idea.

**Comment by ebraker** [comment]
People at Code Table Committee seemed to converge on something like this @mkoo and others, what do you think? Thoughts soon since UI change happens tomorrow. Dusty, what and where are other buttons in Arctos? Once I have the list I'll add hex codes to the repo (@mkoo can you remind me where that is?)
![image](https://user-images.githubusercontent.com/11336485/207986933-e98ee9e8-07e3-4635-a349-e266c06cff05.png)

**Comment by Jegelewicz** [comment]
button save: #FBD29B
                       button search: #CCCC99
                        button link: #99CCFF
                        button clear: #FFCC00
                        button insert: #99CCCC
                        button pic: #CCCC99
                        button delete: #FF9966
                        button quit: #FFFF00
                        button control: #99CCFF

**Comment by ebraker** [comment]
@mkoo I just emailed you a bunch of hex codes and a mock ups See if you like them and if you do, add them to our repo (couldn't remember where we put it). It's slide #10 if you want to dynamically play with palette.
![image](https://user-images.githubusercontent.com/11336485/207998117-5b7086a3-98fd-44b3-963e-6d7d6736c4a4.png)

**Comment by campmlc** [comment]
I like this. While we are at it,should we change " My Stuff " to "My
Profile" to be more professional and use the same term as all other
websites?

**Comment by ebraker** [comment]
@dustymc Is there anyway to hide/bypass the brief flashing of the customize table when selecting "everything on" as the "ghost clicking" occurs? I know it is not a bug but it _looks_ buggy to me and not as smooth a transition as we had in the classic form with "show more options" (plus even a brief flash of the giant customize table is scarrry to uninitiated users IMO...).

**Comment by ebraker** [comment]
Ok last iteration. @dustymc , @mkoo (I aslo listed these in the internal wiki css colors):

![image](https://user-images.githubusercontent.com/11336485/208111992-5d452a8d-5260-4a84-b5f2-0c8601df938b.png)
fonts:  #113d64
navbar: #113d64 (gets white text)
[query box outline with 1pt line weight]: #113d64
[category 1 left side header]: #CCD7E9
[category 1 main body]: #DAE3F3
[category 2 left side header]: #DDE4EE
[category 2 main body]: #EDF1F9
links: #116AC4
visited links: #6B9FB1
hover links: #116AC4 (standard internet blue)
button search [submit query?]: #82B8EA
button login: #82B8EA
button all: #F5C03D
button customize: #CCD7E9
button none: #EDF1F9
button save: #E9B2C1
button clear: #F99F67
button link: # 
button insert: #
button pic: #F99F67
button delete: #6B9FB1
button quit: #
button control: #

not sure of locations of these other buttons, but here are  color variants for buttons: CCD7E9 (turquoise/greenish), EDF1F9 (lightest blue), 82B8EA (blue), C9D2DC (slate blue), F5C03D (yellow), F99F67 (orange)

correct logo to use:
![Arctos-logo_blue-01](https://user-images.githubusercontent.com/11336485/208111683-c2e62200-be86-49a9-9856-ec96568d6893.png)

**Comment by dustymc** [comment]
``

button search [submit query?]: #82B8EA
button login: #82B8EA
button all: #F5C03D
button customize: #CCD7E9
button none: #EDF1F9
button save: #E9B2C1
button clear: #F99F67
button link: #
button insert: #
button pic: #F99F67
button delete: #6B9FB1
button quit: #
button control: #
``

I don't know what those are, I don't use them, they're not valid names. I'll clean them up, the buttons I use start at https://github.com/ArctosDB/PG/blob/master/includes/style.css#L1784 / https://github.com/ArctosDB/arctos/issues/5324#issuecomment-1353842649

```
/* BEGIN purpose-coded buttons */
                        	
	/* save changes */
	input.savBtn {
	   	color:#666666;
	   	font-size:10pt;
		font-weight:bold;
		background-color:#FBD29B;
		border:1px solid;
		border-color:#666666;
		 margin:0.4em;
	}
	
	/* customize something */
		input.cstmBtn {
	   	color:#666666;
	   	font-size:10pt;
		font-weight:bold;
		background-color:#E7EBDA;
		border:1px solid;
		border-color:#666666;
		 margin:0.4em;
	}
	

	/* link; go somewhere not terribly scary */
	input.lnkBtn {
		color:#666666;
		font-size:10pt;
		font-weight:bold;
		background-color:#99CCFF;
		border-color:#666666;
		border:1px solid;
		 margin:0.4em;
	}
	
	/* search button: finds something */
	input.schBtn {
		color:#666666;
        font-size:10pt;
		font-weight:bold;
        background-color:#CCCC99;
		border-color:#666666;
		border:1px solid;
		 margin:0.4em;
    }

	/* clears or resets a form */
	input.clrBtn {
		color:#666666;
		font-size:10pt;
		font-weight:bold;
		background-color:#FFCC00;
		border-color:#666666;
		border:1px solid;
		 margin:0.4em;
	}
	
	/* insert: makes something new */
	input.insBtn {
		color:#666666;
        font-size:10pt;
		font-weight:bold;
		background-color:#99CCCC;
        border-color:#666666;
		border:1px solid;
		 margin:0.4em;
	}
	
	/* opens a pick list */
	input.picBtn {
		color:#666666;
		font-size:10pt;
		font-weight:bold;
		background-color:#CCCC99;
		border-color: #666666;
		border:1px solid;
		 margin:0.4em;
	}
	
	/* delete: something dies with clicky */
	input.delBtn {
		color:#666666;
		font-size:10pt;
		font-weight:bold;
		background-color:#FF9966;
		border-color:#666666;
		border:1px solid;
		 margin:0.4em;
	}
	
	/* quit: go somewhere else without saving any changes */
	
	input.qutBtn {
		color:#666666;
		font-size:10pt;
		font-weight:bold;
        background-color:#F1948A; /***#FFFF00***/
		border-color:#666666;
		border:1px solid;
		 margin:0.4em;
	}

```

**Comment by ebraker** [comment]
Ok, in that case:
input.savBtn color:#89b2c1
input.cstmBtn color:#C9D2DC
input.schBtn color:#82B8EA 
input.clrBtn color:#f99f67
input.picBtn color:#f99f67

I still don't know where this buttons are in arctos (what forms?):
input.lnkBtn - does this mean hypertext color?
input.qutBtn
input.delBtn 
input.insBtn 

but this slide communicates all visible buttons on the new search UI:
![image](https://user-images.githubusercontent.com/11336485/208142641-71df0baf-3dd7-4625-8e62-ac0d42776ec7.png)

**Comment by dustymc** [comment]
@ebraker @mkoo can we split the color/css/style discussions off into a dedicated thread? I'm not sure who all might find it down here.

I think you're proposing some new classes of buttons (for eg check all/none), which is fine, just please provide some sort of definition/intended use (and a 3-ish-letter class name if you wish). I'm not sure where the limit/balance it is, but at some point a few meaningfully-styled buttons tend to become so numerous that nobody can keep up and might as well just be default gray. I don't know where we are on the spectrum, I think some new classes are OK, but there's definitely a point of overload and additions should be carefully considered.

The definitions (however weak) are comments in the code I pasted above, eg

```
/* link; go somewhere not terribly scary */
	input.lnkBtn {
```

I'm sure I overuse that one and the scary red delete too, but see above re: overload....

Do feel free to beef up the definitions (you can edit the css or provide a snippet or just let me know) - "use this when ....." and I will endeavor to comply.

>does this mean hypertext color?

I'm not sure (is that even consistent across environments?), but it's https://www.htmlcsscolor.com/hex/99CCFF

> where this buttons are in arctos (what forms?):

Everywhere. lnkBtn for example:

``585 matches across 140 files``


>navbar

That needs pushed off to https://github.com/ArctosDB/internal/issues/211 (if it means what I think it means) - basically nothing in the code of the current header has a future.

**Comment by ebraker** [comment]
`I think you're proposing some new classes of buttons` No. Just look at the screenshot. I'm trying to align the colors with the buttons you and Teresa sent, but I don't know css so I am probably being unhelpful. It is clear in the screenshot what colors go where. And I'm probably wrong about the term 'navbar' - ignore. I just mean that the line under the bear should be #113d64. Sorry.

![image](https://user-images.githubusercontent.com/11336485/208167869-5a4c5a16-0113-4181-97f5-1e61e6efa751.png)

These are the colors so you can copy paste with plain explanations:
#89b2c1 (this should the color of the "all" button in the form that pops up when you click "customize")
#C9D2DC (this is the color of the customize button)
#82B8EA (this is the color of the submit query and login buttons)
#f99f67 (this is the color of the 'clear form' button and 'choose' button)
#EDF1F9 (this is the color of the 'none' button in the form that pops up when you click "customize")
#F5C03D (this is the color of the 'save' button in the form that pops up when you click "customize
#6B9FB1 (visited links)
#116AC4 (hover links)
#113d64 (fonts)
#113d64 (blue tool bar under the bear, gets white text)
#CCD7E9 (categoryA left column/header) 
#DAE3F3 (categoryA main body)
#DDE4EE (categoryB left column/header)
#EDF1F9 (categoryB main body)
whole stripe-y query block should be outlined in #113d64 ith 1pt line weight

**Comment by campmlc** [comment]
Hi all, I was out all day and now on my phone, and the old form is in production. I'm glad for this, as it buys us some time for final comments and to notify the community. Should we send an announcement?
Also, re: comments,is the lime green in the customize form really in our color pallette? Could we switch to one of the blues or yellows?

**Comment by campmlc** [comment]
Here is a poll for next UI task force meeting: https://www.when2meet.com/?18163351-1mUrJ

**Comment by campmlc** [comment]
Please fill out the poll so we can schedule the next UI task force meeting: https://www.when2meet.com/?18163351-1mUrJ
@dustymc @mkoo @Jegelewicz @ebraker

**Comment by mkoo** [comment]
Filled out the poll-- 16th is MLK holiday but if you guys are willing to
meet, I dont have any meetings that day....

**Comment by campmlc** [comment]
Hi all, let's set up two meetings for each Tuesday Jan 10 and Jan 17 at 4-5
pm MST. We may need to make this a standing meeting until UI issue is
resolved?
If necessary, we can also add another meeting Jan 16 if there are people
who can't make the other two. I'll add to calendar.

**Comment by mkoo** [comment]
Mariel, that sounds great.
Thanks!

**Comment by campmlc** [comment]
Hi all,
I've been sick with fever the last 5 days and have now lost my voice, so I don't think I can lead the meeting this afternoon. Can anyone else? 
Some critical UI issues that are breaking Arctos:
#5333 
#5448 
#5329 
[ArctosDB/internal#211](https://github.com/ArctosDB/internal/issues/211)

**Comment by mkoo** [comment]
I hope you feel better soon!
I will be there at 3 PT
looking forward to working on this, MK
